doc/source/upstream-bug-fixes.rst

```diff
@@ -5,21 +5,39 @@
 =============================================================
 
 +-------------------------------------------------------------------------------------------------------------+
+|:Upstream bug: :mysqlbug:`80607` - main.log_tables-big unstable on loaded hosts                              |
+|:Launchpad bug: :bug:`1554043`                                                                               |
+|:Upstream state: Verified (checked on 2016-03-11)                                                            |
+|:Fix Released: :rn:`5.7.11-4`                                                                                |
+|:Upstream fix: N/A                                                                                           |
++-------------------------------------------------------------------------------------------------------------+
+|:Upstream bug: :mysqlbug:`80606` - Audit Log Plugin worker thread may crash on write call writing fewer ...  |
```

Comment:
The upstream bug title is "my_write, my_pwrite no longer safe to call from THD-less server utility threads"